rfhub2/cli/statistics_extractor.py

```diff
@@ -0,0 +1,102 @@
+from collections import Counter
+from datetime import datetime
+from typing import Dict, List
+from xml.etree import ElementTree as et
+
+
+class StatisticsExtractor:
+    def __init__(self, path: str):
+        self.path: str = path
+        self.source_time_format: str = "%Y%m%d %H:%M:%S.%f"
+        self.destination_time_format: str = "%Y-%m-%d %H:%M:%S.%f"
+
+    def compute_statistics(self) -> List[Dict]:
+        """
+        Returns list of dicst with aggregated statistics extracted from single output.xml file
+        """
+        keywords = self.parse_xml_keywords()
+        return self.aggregate_statistics(keywords)
+
+    def aggregate_statistics(self, keywords: List[Dict]) -> List[Dict]:
+        """
+        Returns list of dicts aggregated data grouped by library/keyword combination.
+        """
+        execution_time = self.get_execution_time()
+        statistics = [
+            {
+                "collection": k[0],
+                "keyword": k[1],
+                "execution_time": execution_time,
+                "times_used": v,
+                "total_elapsed": 0,
+                "min_elapsed": 2147483647,

```

Comment:
We should rather use float('inf') as a starting value

Reply:
Done